keyboards/gizmo_engineering/gk6/keymaps/vial/vial.json

```
@@ -0,0 +1,177 @@
+{
+  "name": "Gizmo Engineering GK6",
+  "vendorId": "0x6532",
+  "productId": "0x0001",
+  "lighting": "none",
```

```suggestion
  "lighting": "vialrgb",
```
c.f. the `rules.mk` file. Was this `"none"` because of AVR constraints? I would strongly suggest `#undef`-ing some of the animations enabled at the keyboard level and/or adding `#define VIALRGB_NO_DIRECT` in the keymap's `config.h` file if that's the concern.